### DISCUSSION TITLE
Just wanted to say thanks

### DISCUSSION THREAD

**Discussion by Akselmo** [Prayag2/konsave#85] [General]
Thanks for this utility, it's great.

As a (still very new) KDE developer I change configs and stuff a lot. Sometimes I hop distros. Sometimes I run a specific distro in VM to test it out.

This tool allows me to have all the configs and shortcuts I'm used to to follow me around.

I shared the tool in here as well https://discuss.kde.org/t/konsave-keeps-your-konfigs-safe/2092

Thanks again and keep up the good work!

**Comment by Prayag2** [comment]
Thank you too :D
